01_foregrounds.ipynb

```diff
@@ -107,11 +107,11 @@
     "            dtype=None,\n",
     "            verbose=False,\n",
     "        )\n",
-    "        emission_map[emission_map == hp.UNSEEN] = np.nan\n",
+    "        emission_map[emission_map == 0] = np.nan\n",
```

Comment:
@keskitalo it seems like the input maps now use 0 instead of UNSEEN for unseen pixels, do you confirm? is there a reason?

Reply:
ah no it seems 0 for UNSEEN is just SAT,
LAT has UNSEEN.

Reply:
Apologies for the confusion. `OpFilterBin` uses the built-in TOAST facilities for map domain manipulation and they do not set unhit pixels to `UNSEEN` like `libmadam` does. Only SAT maps are processed with `OpFilterbin`